lib/program-helper.js

```diff
@@ -59,13 +59,17 @@ function fileExists(cmdPath) {
         return false;
     }
 }
-
+/**
+ * @typedef {Object} HelperType
+ * @prop {() => string} getTSCCommand
+ * @prop {() => string[]} resolveTSFiles
+ * @prop {() => Command} getOptions
+ */
+/** @type {HelperType} */
 var Helper = {};
 
 /**
  * Find tsc file executable path
- *
- * @returns {string}
```

Comment:
I am not (yet) convinced to remove the return types. Yes my understanding is that return types can be "implicit" and that they are declared in the Helper type above. But I continue to really like the idea of making the types in JSDoc comments explicit, even for internal functions, like I would do this with JavaDoc comments. This remark applies to multiple places.